File: python/util/type.py

# 哈希
class Type:

  # 转换: string、int、float
  def ToType(type: str, val):
    if type == 'string':
      return str(val)
    elif type == 'int':
      return int(val)
    elif type == 'float':
      return float(val)
    else:
      return val

  # Interface 转 String
  def Strval(val):
    return str(val)

File: php/util/Type.php
<?php
namespace Util;

/* 类型转换 */
class Type {

  /* 转换: string、int、float */
  static function ToType(string $type, $val) {
    switch($type) {
      case 'string':
        return (string)$val;
      case 'int':
        return (int)$val;
      case 'float':
        return (float)$val;
      default :
        return $val;
    }
  }

  /* Interface 转 String */
  static function Strval($val): string {
    return (string)$val;
  }

}